sample-job-handlers/verify-packages-installed.sh

```diff
@@ -8,28 +8,29 @@ packages=$@
 echo "Username: $user"
 echo "Packages to verify: $packages"
 
-if command -v "rpm" > /dev/null; then
+if id "$user" 2>/dev/null && command -v "sudo" > /dev/null; then
   for package in $packages
   do
-    if id "$user" 2>/dev/null && command -v "sudo" > /dev/null; then
-      sudo -u "$user" -n rpm -q "$package"
-    else
-      echo "username or sudo command not found"
-      rpm -q "$package"
+    if command -v "rpm" >/dev/null; then
+      if command sudo -u "$user" -n rpm -q "$package" > /dev/null; then
+        continue
+      fi
+    fi
+    if command -v "dpkg" >/dev/null; then
+      sudo -u "$user" -n dpkg -s "$package"
     fi
   done
-elif command -v "dpkg" > /dev/null; then
-  # Using -s flag instead of -l for dpkg based on https://github.com/bitrise-io/bitrise/issues/433
+else
+  echo "username or sudo command not found"
   for package in $packages
   do
-    if id "$user" 2>/dev/null && command -v "sudo" > /dev/null; then
-      sudo -u "$user" -n dpkg -s "$package"
-    else
-      echo "username or sudo command not found"
+    if command -v "rpm" >/dev/null; then
+      if command rpm -q "$package" > /dev/null; then
+        continue
+      fi
+    fi
+    if command -v "dpkg" >/dev/null; then
       dpkg -s "$package"
     fi
   done
-else
-  >&2 echo "No suitable package manager found"
-  exit 1
-fi;
\ No newline at end of file
+fi;
```

Comment:
what if both `dpkg` and `rpm` is not installed? We should exit with status code 1 in that case? Why did we remove that logic?

Reply:
Missed this as well... As for your other comment if both were installed it was only checking one of them because of the elif. Will fix